templates/_layout.html

```diff
@@ -14,8 +14,8 @@
       <div class="row">
         <div class="p-navigation__banner">
           <div class="p-navigation__logo">
-            <a href="/">
-              <img src="https://assets.ubuntu.com/v1/d45097a4-snapcraft.io-logotype.svg" style="max-width: 150px; display: block" alt="Snapcraft.io">
+            <a href="https://snapcraft.io">
+              <img class="u-float-left" src="https://assets.ubuntu.com/v1/d45097a4-snapcraft.io-logotype.svg" width="148" alt="Snapcraft.io" />
```

Comment:
Using `u-float-left` seemed the only way in Vanilla to force `display: block` on img without adding custom styles.
But feels a bit hacky to me...

Reply:
Yep, I agree. This is the right solution for now. 

I filed [an issue](https://github.com/vanilla-framework/vanilla-framework/issues/1313) about this in Vanilla - please add to it if you feel I missed anything.